octoprint_mrbeam/__init__.py

```diff
@@ -640,77 +640,73 @@ def get_assets(self):
         assets = dict(
             js=[
                 "js/lib/compare-versions.js",
-                "js/helpers/quick_shape_helper.js",
-                "js/helpers/element_helper.js",
-                "js/helpers/debug_rendering_helper.js",
-                "js/helpers/working_area_helper.js",
+                "js/app/helpers/quick-shape.js",
+                "js/app/helpers/working-area.js",
                 "js/lib/potrace.js",
                 "js/lib/jquery.tinycolorpicker.js",
-                "js/lasercutterprofiles.js",
-                "js/mother_viewmodel.js",
-                "js/folder_list_viewmodel.js",
-                "js/mrbeam.js",
-                "js/color_classifier.js",
-                "js/working_area.js",
-                "js/camera.js",
+                "js/app/view-models/lasercutter-profiles.js",
+                "js/app/view-models/mother.js",
+                "js/app/view-models/folder-list.js",
+                "js/app/view-models/mrbeam.js",
+                "js/app/helpers/color-classifier.js",
+                "js/app/view-models/working-area.js",
+                "js/app/view-models/camera.js",
                 "js/lib/snap.svg-min.js",
-                "js/snap_bugfixes.js",
-                "js/snap_helpers.js",
+                "js/app/snap-plugins/bugfixes.js",
+                "js/app/snap-plugins/helpers.js",
                 "js/lib/dxf.js",
-                "js/snap-dxf.js",
-                "js/render_fills.js",
-                "js/path_convert.js",
-                "js/matrix_oven.js",
-                "js/snap_separate.js",
-                "js/unref.js",
-                "js/snap_transform_plugin.js",
-                "js/convert.js",
-                "js/snap_gc_plugin.js",
-                "js/gcode_parser.js",
-                "js/gridify.js",
-                # "js/lib/photobooth_min.js",
-                "js/svg_cleaner.js",
-                "js/loginscreen_viewmodel.js",
-                "js/wizard_acl.js",
-                "js/netconnectd_wrapper.js",
-                "js/lasersaftey_viewmodel.js",
-                "js/ready_to_laser_viewmodel.js",
+                "js/app/snap-plugins/dxf.js",
+                "js/app/snap-plugins/render-fills.js",
+                "js/app/snap-plugins/path-convert.js",
+                "js/app/snap-plugins/matrix-oven.js",
+                "js/app/snap-plugins/separate.js",
+                "js/app/snap-plugins/unref.js",
+                "js/app/snap-plugins/transform.js",
+                "js/app/view-models/convert.js",
+                "js/app/snap-plugins/gcode.js",
+                "js/app/helpers/gcode-parser.js",
+                "js/app/snap-plugins/gridify.js",
+                "js/app/snap-plugins/svg_cleaner.js",
```

Comment:
Should this be `svg-cleaner.js` instead?

Reply:
Fixed